chapters/inheritance.tex

```diff
@@ -418,7 +418,10 @@ \subsection{Merging of Modifications}\label{merging-of-modifications}
 
 \subsection{Single Modification}\label{single-modification}
 
-Two arguments of a modification shall not modify the same element, attribute, or description-string.  When using qualified names the different qualified names starting with the same identifier are merged into one modifier.  If a modifier with a qualified name has the \lstinline!each! or \lstinline!final! prefix, that prefix is only seen as applied to the final part of the name.
+Two arguments of a modification shall not modify the same element, attribute, or description-string.
+When using qualified names the different qualified names starting with the same identifier are merged into one modifier.
+This merged modifier can be described as a purely syntactic rewriting to an equivalent modifier, except in the case of replaceable redeclarations without a constraining type.
```

Comment:
Reading this for the first time without knowing that there will be an example below felt like there is a hole in the explanation. Perhaps adding, a sentence that there is an explanation coming later would help.
```suggestion
This merged modifier can be described as a purely syntactic rewriting to an equivalent modifier, except in the case of replaceable redeclarations without a constraining type.
The latter is described in the example below.
```